sentry-android-integration-tests/sentry-uitest-android/build.gradle.kts

```diff
@@ -33,6 +33,11 @@ android {
         // Determines whether to support View Binding.
         // Note that the viewBinding.enabled property is now deprecated.
         viewBinding = true
+        compose = true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = Config.composeVersion
```

Comment:
Compose is at `1.3.2` already and we use `1.1.1` - we talked about setting up a CI test matrix already, no need to do it now, but could we at least manually smoke-test if our approach still works with 1.3.2?

Reply:
Just tested with `1.3.2` - works fine ✅